index.html

```diff
@@ -246,7 +246,28 @@ <h2 class="media_header">LET'S CONNECT</h2>
         </div>
       </div>
     </section>
+
     </div>
+    <footer class="contact_container">
+      <div>
+        <div class="container flex">
+          <h1 class="feedback">contact me</h1>
+          <p class="msg">if you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding. 
+            I'd love to help with it</p>
+        </div>
+        <div>
+          <form action="#" method="post">
```

Comment:
- [ ] Please add a [formspree](https://formspree.io/) action to the this form tag 👍 . It should have a similar format to `https://formspree.io/f/{form_id}`. Your form tag should then look something like `<form action="https://formspree.io/f/{form_id}" method="post">`

Reply:
Kindly add action link to your form as specified by the previous reviewer.